templates/shared/gallery/bt-gallery.html.twig

```diff
@@ -15,13 +15,13 @@
 
     <div class="d-flex justify-content-start align-items-center flex-wrap" id="{{id}}-bt-gallery">
 
-        {% for i in 0.. (contents|length / offset) %}
+        {% for i in 0.. (contents|length / offset)|round(0, 'ceil') %}
 
-        {% if i == 0 %}
-            {% set offset = count - 1 %}
-        {% else %}
-            {% set offset = count %}
-        {% endif %}
+            {% if i == 0 %}
+                {% set offset = count - 1 %}
+            {% else %}
+                {% set offset = count %}
+            {% endif %}
```

Comment:
Create a new loop to only manage the first three videos, and then let the second loop deal with rest